script.js

```diff
@@ -0,0 +1,74 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const getData = async () => {
+    const response = await fetch("https://jsonplaceholder.typicode.com/users");
+    const users = await response.json();
+    return users;
+  };
+
+  const createTable = async (thNames = ["id", "name", "phone", "website"]) => {
```

Comment:
You're trying to make this function reusable but, you have hard coded values on the cycle 48th line